package.json

```diff
@@ -69,38 +68,45 @@
     "through2": "^3.0.0"
   },
   "devDependencies": {
+    "@types/mocha": "^7.0.2",
```

Comment:
nit: would be great to keep it sorted :)

Reply:
of course, sorted it by `npx sort-package-json` :)